Category.php

```diff
@@ -48,10 +48,9 @@ public static function pointsFor($checkName, $metric)
 
         if ($points && $metric) {
             return $points *= $metric;
```

Comment:
What do you think about also removing this `return`? Seems strange to use mutation, but also a second exit to return the value as part of the mutation, instead of guarding the mutation only and always returning from the same place after.